commands/alias.js

```diff
@@ -0,0 +1,111 @@
+import ora from 'ora';
+import semver from 'semver';
+import Alias from '../classes/alias.js';
+import { logger, getDefaults, getCWD } from '../utils/index.js';
+import { Alias as AliasFormatter } from '../formatters/index.js';
+
+export const command = 'alias [name] [version] [alias]';
+
+export const aliases = ['a'];
+
+export const describe = `Create a semver major alias for a package, NPM package or import map as identified by its name and version. A package with the given name and version must already exist on asset server. Alias should be the semver major part of the package version. Eg. For a package of version 5.4.3, you should use 5 as the alias. Alias type is detected from an eik.json file in the current working directory.`;
```

Comment:
```suggestion
export const describe = `Create or update a semver major alias for a package, NPM package or import map as identified by its name and version. A package with the given name and version must already exist on the Eik server. The alias should be the semver major part of the package version. Eg. for a package of version 5.4.3, you should use 5 as the alias. The alias type (npm, map, package) is detected from eik.json in the current working directory.`;
```

Reply:
Is it accurate that it has to be `eik.json` – does it not read `"eik"` from `package.json`?